### PR TITLE
Update CHANGELOG for v1.10.0 release

### PR DESCRIPTION
**Pull request by ycombinator**
What it says on the tin.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,8 +9,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+## [1.10.0]
```
Comment (andrewkroh) [inline CHANGELOG.md:20]: The links at the bottom need updated.

Comment (andrewkroh) [reply CHANGELOG.md:20]: It looks like 1.9.0 is also missing a link.

Comment (ycombinator) [reply CHANGELOG.md:20]: Ha, I totally didn't see the bottom.  I was wondering how these links were working.  TIL!

Comment (ycombinator) [reply CHANGELOG.md:20]: Done in ed6f64c.

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**